README.md

```diff
@@ -45,7 +45,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: my-wasmcloud-cluster
-data:
+stringData:
```

Comment:
does this change impact the `nats.creds` section below?

Reply:
Yeah, removed the base64 bit

Reply:
Given the format of the nats.creds, we should decide if it needs to be `nats.creds: |` or just base64 encoded string that we could then unpack (I think we should be able to check if it's base64 encoded or not before we store it), wdyt?

Alternatively I suppose we could consider providing an option for secret reference to pull them from, maybe that's a future iteration.